docs/scripts/deploy-docs.sh

```diff
@@ -88,13 +83,9 @@ if [[ -n "${CI:-}" ]]; then
   git config user.email no-reply@kcp.io
 fi
 
-if [[ "${GITHUB_EVENT_NAME:-}" == "pull_request" || "${GITHUB_EVENT_NAME:-}" == "pull_request_target" ]]; then
+if [[ "${GITHUB_EVENT_NAME}" =~ ^pull_request|pull_request_target$ ]]; then
   mike deploy "${MIKE_OPTIONS[@]}" $VERSION
   exit
 fi
 
-mike deploy "${MIKE_OPTIONS[@]}" "${ALIAS_OPTIONS[@]}"
-
-if [ ${#ALIAS_OPTIONS_LATEST[@]} -gt 0 ]; then
-    mike deploy "${MIKE_OPTIONS[@]}" "${ALIAS_OPTIONS_LATEST[@]}"
-fi
+mike deploy "${MIKE_OPTIONS[@]}" "${ALIAS_OPTIONS[@]}" "$VERSION" "${ALIAS_LIST[@]}"
```

Comment:
FYI, I simplified from multiple runs of `mike deploy` to just one. A single invocation of `mike deploy` can be given multiple aliases. Note the dot-dot-dot after "[alias]" in https://pypi.org/project/mike/0.3.4/#building-your-docs . For an example of a successful test, see the deploy-docs invocation at https://github.com/MikeSpreitzer/kcp-edge-mc/actions/runs/8934350045/job/24541156589#step:5:176 and the corresponding `mike deploy` invocation at https://github.com/MikeSpreitzer/kcp-edge-mc/actions/runs/8934350045/job/24541156589#step:5:213 .

Reply:
Thanks @MikeSpreitzer this is indeed how I use `mike`  from the command line.  
I think we also have some `Voodoo`  steps in the mike process that are not needed but we can push in a follow up PRs